docs/Auto-Scaling.md

```diff
@@ -0,0 +1,5 @@
+# Auto-scaling Support
+
+This integration supports Xperience's [Auto-scaling support](https://docs.kentico.com/developers-and-admins/configuration/auto-scaling-support). This means that your application can distribute computing among multiple applications and therefore increase the number of requests the project can handle.
```

Comment:
The doc updates are ok, but let's use a permanent link here for the Auto-scaling support page:
https://docs.kentico.com/x/SI3WCQ